test/unstable/range-domain-binary-ops.chpl

```diff
@@ -19,7 +19,7 @@ writeln();
 writeln("/////////// domain + and - ///////////");
 
 var d1 = {1..3};  //wass needed?
-var d2, d3: domain(string);
+var d2, d3: domain(string, parSafe=false);
 config var str = "xyz", str2 = "s2";
```

Comment:
While there, could you please remove Line 21 (which I added), assuming the test passes without it.